lms/static/sass/course.scss

```diff
@@ -1,15 +0,0 @@
-/*
```

Comment:
@ormsbee - as far as I can tell we aren't doing anything special with course / course-rtl, so delete away...